tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build (provisioning.k3s && p0) || validation || (provisioning.k3s && !release && !release.oob && !short)
```

Comment:
will we have documentation on when to consider test p0 vs short vs ... ? ). Just looking at this from a perspective of future tests, how will a given person know when it is appropriate to add a tag.

Reply:
We will for sure. I chose these flags because of how often we want to run then, in this case we're running p0 daily. I think for provisioning it has a bit more pull, because these tests are longer running.